.travis.yml

```diff
@@ -19,3 +19,13 @@ notifications:
     secure: L4jZaDmU/D4koIjWfL1D6vsc6eh6N4Qlhwgdf3ymjwK7KB2e66ekxIeuMvBRIWtu8bYnpdkqT+RSR2dNW6i0Qw78stDYy7i118Zxbnd3xlLOK6ClXElYFIcoMaFVlOS1wGRLH++0QP4WfQx8YY7qKrH0RLkmZeTiYDT83G7Mg+Inuf4A8adh51fGtOjzMJ86nugkEFpkad+t/yQI7zOb/bwyoYCr0JR1GmgcBUssgJq+MhjmBJd7deATIiIUCpm3kMBqWyfBkS++pnb4d/9AkMZqTajiYV6x121YUU2Nz5Avf8xGJgLOJP9eycNr7VkLePJv7w/a0+YIissVXLMbj2xJNunThy+5vV+Pp6ifEK/lL7eGcvY1u5wvqKuzO226WFSIq7A4pwbB+3dFv1SYtXqvv67Wpi4i0C5jkVauFIBpzXKZfyBsy3WTTq7AX0Seu6rtSpC/jGbuZIo6GBQnvEjWsl0raqDb9gh+/g6WG898tawlVcUiKsixitYODpWG5kdbB5AbJYjTY24krZeySj7pMKMzsXJJ7ZSCUKtJBpieFqo1bu3DMI1m7ZDDVwA2y8o1n8FsLAwLI17By5huiACvy7kQnahecycUKr8o7yZdTmjUKTxNJ5lxKsS1bTYjjGPOj7DufxTEUTGRVgMkMy+z2/tCiYCL7Z2+92jNv6c=
   on_success: change
   on_failure: always
+deploy:
+  - provider: script
+    script: "./travis-deploy-azure.sh govuk-dfe-bat-deployment-user:$AZURE_WA_PASSWORD bat-dev-search-and-compare-ui-app"
```

Comment:
is $AZURE_WA_PASSWORD set via `travis encrypt`? Just asking b/c I was expecting a new `secure` in the diff for this (unless the `secure` immediately above is AZURE_WA_PASSWORD)

Reply:
I've set it via the user interface at https://travis-ci.org/DFE-Digital/search-and-compare-ui-tests/settings, basically accomplishes the same thing.